packages/cyberstorm/src/components/Tag.tsx

```diff
@@ -19,13 +19,21 @@ export const Tag: React.FC<TagProps> = (props) => {
     <div
       /* TS is not aware of defaultProps of function components. */
       /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
-      className={`${styles.tag} ${styles[tagStyle!]} ${styles[tagSize!]}`}
+      className={`${styles.root} ${getStyle(tagStyle)} ${styles[tagSize!]}`}
     >
-      <div className={styles.tagIcon}>{leftIcon}</div>
-      {label ? <div className={styles.tagLabel}>{label}</div> : null}
-      <div className={styles.tagIcon}>{rightIcon}</div>
+      <div className={styles.icon}>{leftIcon}</div>
```

Comment:
The icon wrappers should be omitted if no icon is given. Otherwise `gap` will think there needs to be a gap between it and an adjacent item, thus displaying an extra whitespace.

Reply:
Added conditions for both icons